### PR TITLE
Dev/wl 8.3

### PR DESCRIPTION
**Pull request by wlchn**

### CODE REVIEW

File: frontend/src/forms/Route/index.tsx [hunk 1]
```diff
@@ -459,6 +471,11 @@ class RouteFormRaw extends React.PureComponent<Props, State> {
                       <Alert className="alert" severity="info">
                         {sc.ROUTE_HTTPS_ALERT}
                       </Alert>
+                      {includesForceHttpsDomain(hosts) ? (
+                        <Alert className="alert" severity="warning">
+                          The .dev and .app top-level domains is included on the HSTS preload list, HTTPS is required.
```
Comment (iAladdin) [inline frontend/src/forms/Route/index.tsx:476]: please move this line string  to `stringConstants`

Comment (iAladdin) [reply frontend/src/forms/Route/index.tsx:476]: @wlchn

File: frontend/src/utils/domain.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import Immutable from "immutable";
+
+export const forceHttpsDomainSuffixes = [".dev", ".app"];
```
Comment (iAladdin) [inline frontend/src/utils/domain.ts:3]: Please add more TLDs,the full list from: 
https://source.chromium.org/chromium/chromium/src/+/master:net/http/transport_security_state_static.json

```
  { "name": "android", "policy": "public-suffix", "mode": "force-https", "include_subdomains": true },
    { "name": "app", "policy": "public-suffix", "mode": "force-https", "include_subdomains": true },
    { "name": "bank", "policy": "public-suffix", "mode": "force-https", "include_subdomains": true },
    { "name": "chrome", "policy": "public-suffix", "mode": "force-https", "include_subdomains": true },
    { "name": "dev", "policy": "public-suffix", "mode": "force-https", "include_subdomains": true },
    { "name": "foo", "policy": "public-suffix", "mode": "force-https", "include_subdomains": true },
    { "name": "gle", "policy": "public-suffix", "mode": "force-https", "include_subdomains": true },
    { "name": "gmail", "policy": "public-suffix", "mode": "force-https", "include_subdomains": true },
    { "name": "google", "policy": "public-suffix", "mode": "force-https", "include_subdomains": true, "pins": "google" },
    { "name": "hangout", "policy": "public-suffix", "mode": "force-https", "include_subdomains": true, "pins": "google" },
    { "name": "insurance", "policy": "public-suffix", "mode": "force-https", "include_subdomains": true },
    { "name": "meet", "policy": "public-suffix", "mode": "force-https", "include_subdomains": true },
    { "name": "new", "policy": "public-suffix", "mode": "force-https", "include_subdomains": true },
    { "name": "page", "policy": "public-suffix", "mode": "force-https", "include_subdomains": true },
    { "name": "play", "policy": "public-suffix", "mode": "force-https", "include_subdomains": true },
    { "name": "search", "policy": "public-suffix", "mode": "force-https", "include_subdomains": true },
    { "name": "youtube", "policy": "public-suffix", "mode": "force-https", "include_subdomains": true },
    // Domains on the public suffix list that have requested to be preloaded.
```

Comment (wlchn) [reply frontend/src/utils/domain.ts:3]: 👌🏻 Got it

**Pull request merged**